### ISSUE TITLE
[Bug] Local Image is not working in AvatarView

### ISSUE THREAD

**Issue by QwertyyD** [xamarin/XamarinCommunityToolkit#1025]
### Description
When AvatarView is used with LocalImage it displays Text value instead of image, while the same image is displayed in Xamarin.Forms.Image control.
### Steps to Reproduce

Sample : 
```
<xct:AvatarView Text="Avatar" Source="tab_about"/>
<Image Source="tab_about"/>
```

### Expected Behavior
Image should be displayed.
### Actual Behavior
Text value is displayed.
### Basic Information

- Version with issue: 1.0.3
- Last known good version:
- IDE: VS 16.7.8

**Comment by AndreiMisiukevich** [comment]
@QwertyyD can you specify the platform?

**Comment by AndreiMisiukevich** [comment]
Fixed in #1042 will be available in the next major release (v1.1)
